src/blog/2024/05/why-you-need-a-low-code-platform.md

```diff
@@ -5,14 +5,11 @@ description: Discover how low-code platforms like FlowFuse empower domain expert
 date: 2024-05-22
 authors: ["grey-dziuba"]
 image: /blog/2024/05/images/why-you-need-a-low-code-platform.png
+keywords: low-code, digital Transformation, app development, Citizen Development 
 tags:
     - posts
```

Comment:
Let's take the chance to remove all the `post` or `posts` tags
```suggestion
```

Reply:
@sumitshinde-84 Can you pick this up?